SteamProject/MySteam/Category.h

```diff
@@ -15,19 +14,24 @@ class FCategory
 
     FCategory();
 
-    FCategory(const std::string NewCategoryName);
+    FCategory(const std::string& NewCategoryName);
+
+    FCategory(const FCategory &InCategory);
```

Comment:
Careful there with your formatting, you want to keep consistency. We use references immediately after the type with no spaces.